tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/ssd.py

```diff
@@ -449,3 +449,110 @@ def process(self, raw, identifiers, frame_meta):
                                               detections['x_mins'], detections['y_mins'], detections['x_maxs'],
                                               detections['y_maxs']))
         return result
+
+
+class SSDTFAdapter(Adapter):
+    __provider__ = 'ssd_tf'
+
+    @classmethod
+    def parameters(cls):
+        params = super().parameters()
+        params.update({
+            'scores_out': StringField(description='scores output'),
+            'boxes_out': StringField(description='boxes output'),
+            'confidence_threshold': NumberField(optional=True, default=0.05, description="confidence threshold"),
+            'iou_threshold': NumberField(optional=True, default=0.5, description="IoU threshold"),
+            'keep_top_k': NumberField(optional=True, value_type=int, default=200, description="keep top K")
+        })
+        return params
+
+    def configure(self):
+        self.scores_out = self.get_value_from_config('scores_out')
+        self.boxes_out = self.get_value_from_config('boxes_out')
+        self.confidence_threshold = self.get_value_from_config('confidence_threshold')
+        self.iou_threshold = self.get_value_from_config('iou_threshold')
+        self.keep_top_k = self.get_value_from_config('keep_top_k')
+        self.outputs_verified = False
+
+    def select_output_blob(self, outputs):
+        self.scores_out = self.check_output_name(self.scores_out, outputs)
+        self.boxes_out = self.check_output_name(self.boxes_out, outputs)
+        self.outputs_verified = True
+
+    @staticmethod
+    def calc_iou(target, candidates):
+        target_tiled = np.tile(target[np.newaxis, :], (candidates.shape[0], 1))
+
+        left_top = np.maximum(target_tiled[:, :2], candidates[:, :2])
+        right_bottom = np.minimum(target_tiled[:, 2:], candidates[:, 2:])
+
+        delta = np.maximum(right_bottom - left_top, 0)
+
+        intersect = delta[:, 0] * delta[:, 1]
+
+        diff = target_tiled[:, 2:] - candidates[:, :2]
+        area = diff[:, 0] * diff[:, 1]
+
+        iou = intersect / (2 * area - intersect)
+        return iou
+
+    def process(self, raw, identifiers, frame_meta):
+        result = []
+        raw_output = self._extract_predictions(raw, frame_meta)
+        if not self.outputs_verified:
+            self.select_output_blob(raw_output)
+
+        for identifier, logits, boxes in zip(identifiers, raw_output[self.scores_out], raw_output[self.boxes_out]):
+            detections = {'labels': [], 'scores': [], 'x_mins': [], 'y_mins': [], 'x_maxs': [], 'y_maxs': []}
```

Comment:
looks like this structure is useless (you obtain boxes, labels and scores once for single image, no need to repack them to dictionary. you can directly use variables for prediction class initialization